onos-config/values.yaml

```diff
@@ -35,6 +35,14 @@ debug: false
 
 devices: []
 
+# model plugins list
+# deployed as a sidecar to onos-config, eg:
+
+# modelPlugins:
```

Comment:
Should we have some default model plugins here? 

@teone  @tomikazi

Reply:
I use these by default:
modelPlugins:
  - name: devicesim1
    image: onosproject/devicesim:1.0.0
    port: 5152
  - name: testdevice1
    image: onosproject/testdevice:1.0.0
    port: 5153
  - name: testdevice2
    image: onosproject/testdevice:2.0.0
    port: 5154

Reply:
yeah we should put them there

Reply:
Done